cypress/support/cypress-support/src/main.js

```diff
@@ -253,14 +254,36 @@ export default function (module) {
   Cypress.Commands.add('sendMessagetoPlatforms', (requestMap) => {
     cy.wrap(requestMap, { timeout: CONSTANTS.SEVEN_SECONDS_TIMEOUT }).then(async (requestMap) => {
       return new Promise(async (resolve) => {
-        const message = await config.getRequestOverride(requestMap);
-        // perform MTC call/FB call only if the message is not null
-        if (message != null) {
-          const response = await transport.sendMessage(message);
-          const result = config.getResponseOverride(response);
-          resolve(result);
+        const [moduleName, methodName] = requestMap.method.split('.');
+        Cypress.env(CONSTANTS.REQUEST_OVERRIDE_METHOD, methodName);
+        // Check if request is for fcs setters
+        if (moduleName === 'fcsSetter') {
+          const method = fcsSetter[methodName];
+          if (method && typeof method === 'function') {
+            try {
+              const { attribute, value } = requestMap.params;
+              const response = await method(attribute, value);
+              resolve(response);
+            } catch (error) {
+              resolve(
```

Comment:
We don't want to resolve here. Whatever error we get here (Probably from a setterFailure or setterNotImplemented) should bubble up and fail the step.

Calling setterFailure() again here is redundant, especially if we got here because of a setterNotImplemented()